CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing to dash
+
+## Getting Started
+
+Refer to the [readme](README.md) for installation and development instructions.
+
+## Coding Style
+
+Please lint any additions to react components with `pylint` and `flake8`.
+
+## Pull Request Guidelines
+
+Use the [GitHub flow][] when proposing contributions to this repository (i.e. create a feature branch and submit a PR against the master branch).
+
+## Running the Tests
+
+**Warning:** _Tests do not currently run on windows._
+
+To run the tests, you can use Python's `unittest` module, or a test runner like `nose2`.
+
+To run all of the tests:
+`python -m unittest tests`
+
+Or:
+`nose2 -v`
+
+## Local configuration
+You can configure the test server with the following variables:
+### DASH_TEST_CHROMEPATH
+If you run a special chrome set the path to your chrome binary with this environment variable.
+
+### DASH_TEST_PROCESSES
+If you encounter errors about Multi-server + Multi-processing when running under Python 3 try running the tests with the number of server processes set to 1.
+
+### Example: single test run with configuration
+```
+DASH_TEST_CHROMEPATH=/bin/google-chrome-beta DASH_TEST_PROCESSES=1 python -m unittest -v test.test_integration.Tests.test_inputs
+```
+
+## Making a contribution
+
+1. Create a pull request and tag the Plotly team (`@plotly/dash`) and tag / request review from [@rmarren1](https://github.com/rmarren1) and [@T4rk1n](https://github.com/T4rk1n).
```

Comment:
Could we add a step like: "For larger features, you're contribution will have a higher likelihood of getting merged into this repository if you create an issue to discuss the changed that you'd like to make before you create a pull request."